README.md

```diff
@@ -154,6 +154,28 @@ then 4 times in a row and issue an appropriate alert.
 
 ## Running a test environment
 
+### Kwok Container
+
+To run the Kwok container with the Kwok Kubernetes tool, follow these steps:
+
+1. Bring up the Kwok cluster by running the following command:
+
+`podman run --rm -d -p 8080:8080 --name kwok-cluster registry.k8s.io/kwok/cluster:v0.3.0-k8s.v1.27.3`
+
+2. Once the Kwok cluster is up and running, set the cluster details in the OpenShift client with the following command:
+
+`oc config set-cluster kwok --server=http://127.0.0.1:8080`
+
+Now we can access the cluster using kubectl cmds.
```

Comment:
this sentence probably appears too soon....

Reply:
once we set the cluster it also sets the context from kubeconfig. no need to change it manually

Reply:
Updated